bb

```diff
@@ -297,8 +299,14 @@ bb_courses() {
 		# term=2012FALL
 		# crn=77613
 		# title='THE SCI OF DATA STRUCTURES'
+		term="$(cut -c 5 <<< $term)$(cut -c 6- <<< $term | tr \
+			'[:upper:]' '[:lower:]') $(cut -c -4 <<< $term)"
+		if [[ "$term" != "$current_term" ]]; then
+			echo "$term"
+			current_term="$term"
+		fi
 
-		echo $name.$term.$crn - $title
+		printf "	%5s %-8s -  %s\n" "$crn" "$(sed 's/^[A-Z]\+/\0 /' <<< $name)" "$title"
```

Comment:
OS X's sed doesn't support `\+` or `\0`. I suggest changing it to `s/^[A-Z]*/& /`